modules/orderService.js

```diff
@@ -89,7 +89,7 @@ module.exports.addOrder = (newSID, newAdd, newStatus, newTotal) => {
 
 module.exports.UpdateOrder = (OrdId, newStatus) => {
 	return new Promise((resolve, reject) => {
-		Products.updateOne({ _id: OrdId }, { status: newStatus }, (err, result) => {
+		Products.updateOne({ _id: OrdId }, { status: newStatus, updated_at: Date.now }, (err, result) => {
```

Comment:
I could be wrong, but Date.now is just the reference to the function now(). Probably want to change this to Date.now()

Reply:
![image](https://user-images.githubusercontent.com/35984346/76689054-e2db0f00-6608-11ea-9e3a-3e5f4991ce66.png)

Reply:
Date.now is how it's declared.